index.html

```diff
@@ -12,10 +12,27 @@
       align-items: center;
       flex-direction: column;
     }
+
+    .hidden {
+      display: none;
+    }
+
+    .inline-text {
+      display: inline-block;
+      position: fixed;
+      top: 0;
+      right: 0;
+      margin-right: 1em;
+      background-color: #FFF3CD;
+      padding: 10px;
+      color: #B3994E;
+      z-index: 1;
+    }
   </style>
 </head>
 
 <body>
+  <p id="notify-clipboard-text" class="hidden"></p>
```

Comment:
```suggestion
  <p id="notify-clipboard-text" class="notification hidden"></p>
```

Reply:
Solved. Also used div to avoid margin top being set